tests/test_mpx_inner.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+This tests the refactored C interface inner_self through its Python wrapper.
+Tests were adapted from the general mpx tests. Missing components are computed inline.
+
+"""
+import os
+import numpy as np
+import matrixprofile
+from matrixprofile.algorithms import cympx_inner
+from matrixprofile import cycore
+MODULE_PATH = matrixprofile.__path__[0]
+
+
+def dfdg_with_verif(ts, mu, test_offsets):
```

Comment:
It is convention to prefix unit tests with test_. It is also a requirement for pytest to execute the function as a unit test. I will be surprised if pytest is executing these as you expect.

Reply:
Oh. That wasn't run on its own. I ran crosscov and the difference equation tests as sub-components of a larger test. The larger test fails at that stage if either does not pass. 

Would you prefer that I break these out? If so, how would you like to set up reference data? This one in particular measures the difference between two co-moment calculations. It may look different from an analytic standpoint, but that is its purpose.

Reply:
Leaving it as is is fine. Please take a moment to clarify the intent of the functions by commenting in the code.